hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java

```diff
@@ -1414,7 +1414,7 @@ private static X509CertificateHolder generateX509CertHolder(
         .setSubject(subject)
         .setDigitalSignature(true)
         .setDigitalEncryption(true);
-    
+
```

Comment:
Please revert this unnecessary change, which is the only change of this file.

Reply:
+1

Reply:
Hi @ChenSammi I am not sure why this is coming as I dont see any local changes that I have made to this class 
<img width="644" alt="image" src="https://github.com/apache/ozone/assets/98023601/3c591b82-52ff-4d65-b019-ffc3800749c8">